tests/cases/fourslash/refactorConvertParamsToDestructuredObject_templateLiteral.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+////function /*a*/insert/*b*/(template: string, overwriteBefore = 0) {}
+////insert(`this is \${not} a substitution`);
+
+
+goTo.select("a", "b");
+edit.applyRefactor({
+    refactorName: "Convert parameters to destructured object",
+    actionName: "Convert parameters to destructured object",
+    actionDescription: "Convert parameters to destructured object",
+    newContent: [
+      'function insert({ template, overwriteBefore = 0 }: { template: string; overwriteBefore?: number; }) {}',
+      'insert({ template: `this is \\${not} a substitution` });'
```

Comment:
Don't understand why it becomes `\\$` instead of `\$`

Reply:
`\\` is just an escape for `\` in the actual fourslash code 😄

Reply:
Video evidence for clarification 🙃:
 
![escape](https://user-images.githubusercontent.com/3277153/62403385-9b07eb80-b541-11e9-8a37-7fd5a7e2f16b.gif)